src/main/java/raphael/Raphael.java

```diff
@@ -0,0 +1,71 @@
+package raphael;
+
+import raphael.command.Command;
+import raphael.parser.Parser;
+import raphael.storage.Storage;
+import raphael.task.TaskList;
+import raphael.ui.Ui;
+
+/**
+ * The main class of Raphael.
+ */
+public class Raphael {
+    public static final String LOGO = "\n"
+            + "  _____                _                   _\n"
+            + " |  __ \\              | |                 | |\n"
+            + " | |__) | __ _  _ __  | |__    __ _   ___ | |\n"
+            + " |  _  / / _` || '_ \\ | '_ \\  / _` | / _ \\| |\n"
+            + " | | \\ \\| (_| || |_) || | | || (_| ||  __/| |\n"
+            + " |_|  \\_\\\\__,_|| .__/ |_| |_| \\__,_| \\___||_|\n"
+            + "               | |\n"
+            + "               |_|\n";
+    public static final String BOT_NAME = "raphael";
```

Comment:
I like the naming as it conforms to the coding standard, as it is already confirmed here.